components/sidebar/more_public_direct_channels.jsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Modal, Tabs, Tab} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import MorePublicChannels from 'components/more_channels';
+import MoreDirectMessages from 'components/more_direct_channels';
+
+import {localizeMessage} from 'utils/utils';
+
+export default class MorePublicDirectChannels extends React.PureComponent {
+    static propTypes = {
+        handleNewChannel: PropTypes.func,
+        onModalDismissed: PropTypes.func,
+    };
+
+    state = {
+        show: true,
+        key: 'channels',
+    };
+
+    handleHide = () => {
+        this.setState({show: false});
+    };
+
+    handleExit = () => {
+        if (this.props.onModalDismissed) {
+            this.props.onModalDismissed();
+        }
+    };
+
+    handleSelect = (key) => {
+        this.setState({key});
+    }
+
+    render() {
+        return (
+            <Modal
+                dialogClassName={'more-modal more-direct-channels more-public-direct-channels'}
+                show={this.state.show}
+                onHide={this.handleHide}
+                onExited={this.handleExit}
+            >
+                <Modal.Header closeButton={true}>
+                    <Modal.Title>
+                        <FormattedMessage
+                            id='more_public_direct_channels.title'
+                            defaultMessage='Channels and Direct Messages'
+                        />
+                    </Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <Tabs
+                        className='modal-tabs'
+                        defaultActiveKey='channels'
+                        activeKey={this.state.key}
+                        onSelect={this.handleSelect}
+                    >
+                        <Tab
```

Comment:
Can you add `id`s to each of these `Tab`s? React-bootstrap wants them for accessibility
![image](https://user-images.githubusercontent.com/3277310/47751939-87a1ba00-dc69-11e8-8fb8-67acb67e8233.png)